label_studio/data_import/uploader.py

```diff
@@ -74,9 +74,9 @@ def create_file_upload(user, project, file):
     if settings.SVG_SECURITY_CLEANUP:
         content_type, encoding = mimetypes.guess_type(str(instance.file.name))
         if content_type in ['image/svg+xml']:
-            clean_xml = allowlist_svg(instance.file.read())
+            clean_xml = allowlist_svg(instance.file.read().decode())
             instance.file.seek(0)
-            instance.file.write(clean_xml)
+            instance.file.write(clean_xml.encode())
```

Comment:
These changes were needed because lxml's `clean_html` seems to no longer work on bytes-like objects in recent versions. I hope it's okay to decode and encode the svg string?

Reply:
tested locally and seems to work right - only real concern area I could think of would be giant SVG uploads that are expensive to decode, but that seems like it'd cause other problems anyway.